packages/core/bootstrap/src/lib/cache/index.ts

```diff
@@ -141,9 +156,10 @@ export const withCache: Middleware = async (execute, options = defaultOptions())
     const coalescingKey = _getCoalescingKey(key)
     const endMetrics = metrics.beginObserveCacheExecutionDuration({
       participantId: key,
-      feedId: data.debug?.feedId,
+      feedId: data.debug?.feedId || 'unknown',
```

Comment:
I think we should keep it as undefined, i feel like there's a couple other places that are marked as `undefined`, would be a good ticket to make it so that we unify all unknown values in metrics